adds readme with wireframes

JMarcussen opened the pull request.

JMarcussen committed:
adds readme with wireframes

JMarcussen committed:
adds missing wireframes

JMarcussen commented:
Hi @axamei , I believe I have completed the assignment, however the images are not showing when I preview. Is there a way I can fix this? Thanks!

axamei commented:
@JMarcussen The format of the image reference needs to change. Change to include the folder, the curly braces should be replaced by the square brackets, and no space between the square bracket and parentheses, e.g., `![Wireframe of Home Page](wireframes/Wireframes-examples-01.jpg)`

axamei commented on README.md:
```diff
@@ -1,2 +1,39 @@
+
+This is where I will include visual wireframes in order to organize my portfolio site.
+
+!{Wireframe of Home Page} (Wireframes-example-01.jpg)
```
Change to `![Wireframe of Home Page](wireframes/Wireframes-examples-01.jpg)`

axamei commented on README.md:
```diff
@@ -1,2 +1,39 @@
+Footer - will include copyright info and contact info
+
+
+!{Wireframe of Work Page} (Wireframes-example-02.jpg)
```
Change to correct format and include folder

axamei commented on README.md:
```diff
@@ -1,2 +1,39 @@
+Footer - will include copyright info and contact info
+
+
+!{Wireframe of About Page} (Wireframes-example-03.jpg)
```
Change to correct format and include folder

axamei commented:
@JMarcussen Please make sure these changes are made before Thursday.

axamei commented:
@JMarcussen I still don't see the requested changes in this branch. Please reach out to me.

JMarcussen commented:
@axamei do I create a new pull request now that the changes have been made?

axamei commented:
@JMarcussen make a commit with the requested changes and then push the branch. It should show up here.

JMarcussen committed:
Correct location and format

JMarcussen commented:
All set. Made a commit, and then pushed the branch.

JMarcussen committed:
Added Web pages

Added 3 HTML web pages, simple navigation, inserted heading levels, insert graphic, insert unordered list, validate pages

JMarcussen committed:
Changes saved
